lib/cylc/batch_sys_handlers/loadleveler.py

```diff
@@ -55,7 +55,7 @@ def format_directives(self, job_conf):
             directives["wall_clock_limit"] = "%d,%d" % (
                 job_conf["execution_time_limit"] + 60,
                 job_conf["execution_time_limit"])
-        for key, value in job_conf["directives"].items():
+        for key, value in list(job_conf["directives"].items()):
```

Comment:
Is this list( statement really needed?

Reply:
It might be, there appears to be a circular nature to the `directives` variable, for safety I left the `list` in:

```python
directives = job_conf["directives"].__class__()

for key, value in list(job_conf["directives"].items():
    directives[key] = value
```

Reply:
Fair Play.